setup.py

```diff
@@ -1,5 +1,7 @@
 from setuptools import setup, find_packages
 
+with open('requirements.txt', 'r') as f:
+    INSTALL_REQUIRES = f.readlines()
```

Comment:
This will include `\n` at the end of the line for each line, I think it works but it's not clean. Maybe you can do something like:

```python
INSTALL_REQUIRES = f.read().splitlines()
```